chalk-engine/Cargo.toml

```
@@ -18,3 +18,7 @@ tracing = "0.1"
 
 chalk-derive = { version = "0.15.0-dev.0", path = "../chalk-derive" }
 chalk-ir = { version = "0.15.0-dev.0", path = "../chalk-ir" }
+chalk-solve = { version = "0.15.0-dev.0", path = "../chalk-solve" }
+
+[dev-dependencies]
+chalk-integration = { path = "../chalk-integration" }
```

```suggestion
chalk-integration = { version = "0.15.0-dev.0", path = "../chalk-integration" }
```

I don't know if using paths only for dev dependecies also prevents publish.

---

I copied this from the `Cargo.toml` for `chalk-solve`. This is fine :) Thanks though.